try{ 
	var a = 1
	console.log(a)
} catch(e){
	console.log(e)
}

console.log(a)
